root.tf

```diff
@@ -185,6 +189,8 @@ module "prod_specific_permissions" {
   read_terraform_state_policy_arn = module.terraform_permissions.read_terraform_state_policy_arn
   terraform_state_lock_access_arn = module.terraform_permissions.terraform_state_lock_access_arn
   terraform_describe_account_arn  = module.terraform_permissions.terraform_describe_account_arn
+  release_bucket_arn              = module.release_artefacts_s3.s3_bucket_arn
+  staging_bucket_arn              = module.staging_artefacts_s3.s3_bucket_arn
```

Comment:
What's the reason for having an artifact bucket in every environment? Are there environment-specific things that we need with each release? I'd have expected there to be just one bucket for this (maybe in the management account?) but I haven't though through all the different libraries that we're releasing. 🙂

Reply:
There is only one bucket in the management account but we're creating three roles, one for each environment which actually isn't needed because they all say the same thing and they're only accessed by Jenkins anyway. I've changed it so it only creates on role and updated Jenkins to use it.

Reply:
Ah, OK! I misread the bucket config, sorry. Thanks for updating the permissions. 👍